pages/report/index.js

```diff
@@ -0,0 +1,53 @@
+import dynamic from "next/dynamic";
+import Image from "next/image";
+import Link from "next/link";
+import styled from "styled-components";
+const BarChartPage = dynamic(() => import("@/components/Chart"), {
```

Comment:
Her you are importing your bar component so I assume if under certain conditions the loading takes some time maybe you would want t include a loading message.

```suggestion
const BarChartPage = dynamic(() => import("@/components/Chart"), {
ssr: false,
  loading: () => <p>Loading chart...</p>
```